### PR TITLE
Fix minor things: rand.Seed, golint, etc.

### PR DESCRIPTION
**Pull request by plutov**
Detailed explanation I put into comment.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -20,12 +20,11 @@ func main() {
 
 	masTree := tree.NewTree(40, company)
 
-	if live {
-		for true {
-			masTree.Print()
-			time.Sleep(2 * time.Second)
-		}
-	} else {
+	for true {
```
Comment (plutov) [inline main.go:23]: 8 -> 6 lines. removed else.

File: tree/color.go [hunk 2]
```diff
@@ -21,14 +23,5 @@ var (
 )
 
 func randomColor() *color.Color {
-	rand.Seed(time.Now().UnixNano())
 	return colors[rand.Intn(colorSize)]
 }
-
-func yellowColor() *color.Color {
```
Comment (plutov) [inline tree/color.go:28]: colors[1] can be painful if someone will change the order of colors.

File: tree/color.go [hunk 3]
```diff
@@ -21,14 +23,5 @@ var (
 )
 
 func randomColor() *color.Color {
-	rand.Seed(time.Now().UnixNano())
```
Comment (plutov) [inline tree/color.go:24]: Seeding should be done exactly 1 time before using rand. Here Seed can be called multiple times, also in leaf.go. So I moved rand.Seed to init() func of leaf.go.

File: tree/tree.go [hunk 4]
```diff
@@ -2,12 +2,13 @@ package tree
 
 import "fmt"
 
+// Tree definition
```
Comment (plutov) [inline tree/tree.go:5]: To fix golint warnings.

File: tree/tree.go [hunk 5]
```diff
@@ -27,37 +28,37 @@ func NewTree(width int, company string) *Tree {
 		}
 	}
 
-	leaves = append(leaves, bottomLeaves(width, "^"))
-	leaves = append(leaves, bottomLeaves(width, " "))
+	leaves = append(leaves, bottomLeaves(width, "^"), bottomLeaves(width, " "))
```
Comment (plutov) [inline tree/tree.go:31]: Not sure if you'll like it :)

File: tree/tree.go [hunk 6]
```diff
@@ -27,37 +28,37 @@ func NewTree(width int, company string) *Tree {
...
 			case "★":
-				yellowColor().Print(leaf)
+				yellowColor.Print(leaf)
 			default:
 				randomColor().Print(leaf)
 			}
 		}
 
-		fmt.Print("\n")
+		fmt.Println()
```
Comment (plutov) [inline tree/tree.go:53]: As we use everywhere Println I think we should use it here also.

File: main.go (hunk 1 above)
Comment (songjiayang) [reply main.go:23]: Look good, but I really don't like do check in every loop.

File: tree/color.go (hunk 2 above)
Comment (songjiayang) [reply tree/color.go:28]: you are right.

File: tree/color.go (hunk 3 above)
Comment (songjiayang) [reply tree/color.go:24]: I want to random color every time, are you sure it work?

File: tree/tree.go (hunk 5 above)
Comment (songjiayang) [reply tree/tree.go:31]: I like it.

**Review by songjiayang — APPROVED** [review]
LGTM

Hi @plutov I do a test, your code work well, thank you so much. 

And I want to say your code style look so good. 😄

File: tree/color.go (hunk 3 above)
Comment (songjiayang) [reply tree/color.go:24]: It work.

Comment (plutov) [reply tree/color.go:24]: That's the point, documentation says that we need to seed only 1 time.

**Pull request merged**